pkg/client_data/lib/account_api.dart

```diff
@@ -49,6 +49,32 @@ class ClientSessionStatus {
   bool get isValid => expires != null && DateTime.now().isBefore(expires);
 }
 
+/// Account-specific information about a likes.
+@JsonSerializable()
+class PackageLikes {
+  List<PackageLike> likedPackages;
+  PackageLikes(this.likedPackages);
+
+  factory PackageLikes.fromJson(Map<String, dynamic> json) =>
+      _$PackageLikesFromJson(json);
+
+  Map<String, dynamic> toJson() => _$PackageLikesToJson(this);
+}
+
+@JsonSerializable()
+class PackageLike {
```

Comment:
nit: maybe rename this `PackageLikeStatus` (`PackageLikeResponse` would also work).